andela/AndelaEatsUI
# #161704453 Add date range on view menu page

**What does the PR do?**
- This PR adds `Date Range on View Menu Page`

**Description of Task to be completed?**
- add date range on the view menu page
- consume real API
- refactor fetch action with the date range

**How should this be manually tested?**
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `ft-add-date-range-on-view-menu-161704453`
- run these commands `npm run bundle` and `npm run dev` 
- Log in as an `Admin`, navigate to `admin dashboard`, select `Menu Icon`, and click on `Add Menu`
- run the command `npm run test-dev` to ensure all tests are passing

**What are the relevant pivotal tracker stories?**
- [#161704453](https://www.pivotaltracker.com/story/show/161704453)

**Screenshots**
<img width="1680" alt="screen shot 2018-11-04 at 7 47 12 pm" src="https://user-images.githubusercontent.com/26305570/47968496-26178c00-e06b-11e8-8546-93f0ca43b7af.png">
<img width="1680" alt="screen shot 2018-11-04 at 7 47 23 pm" src="https://user-images.githubusercontent.com/26305570/47968497-26178c00-e06b-11e8-943c-1eb4c54ba656.png">
